phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
+<phpunit colors="true">
```

Comment:
I have removed "bootstrap" declaration from dist file. 

I use IntelliJ for testing, and it is easy to setup all custom paths - to PhpUnit, to Composer, to phpunit.xml...  So this declaration is not required at all.

Moreover -> I would like to restructure bundle:

-> root
   -> src (bundle code)
   -> tests (tests)
   -> other files, composer.json, readme.md and so on...

And I would like to add 'autoload-dev' declaration in composer.json, see per example: https://github.com/RunOpenCode/assets-injection/blob/master/composer.json